engines/director/detection.h

```diff
@@ -41,6 +42,21 @@ struct DirectorGameDescription {
 	uint16 version;
 };
 
+
+class FallbackBlacklist {
```

Comment:
Since you have `_fallback_blacklisted_names` HashMap, there is zero sense in adding this class. Please remove.